src/dsp_tools/xmllib/helpers.py

```diff
@@ -0,0 +1,61 @@
+import json
+from typing import Any
+from typing import Iterable
+
+
+def create_label_to_name_list_node_mapping(
```

Comment:
I changed the name from the original because I did not think it very clear. What do you think of this @Notheturtle

Reply:
what about `create_label_to_name_mapping_for_lists()`?

Reply:
The PR title still has the old function name, I would update the PR title